samples/v2/subdagio/mixed_parameters.py

```diff
@@ -0,0 +1,48 @@
+import os
+
+from kfp import Client
+from kfp import dsl
+from kfp.compiler import Compiler
+
+
+@dsl.component
+def core_comp() -> int:
+    return 1
+
+
+@dsl.component
+def crust_comp(x: int, y: int):
+    print('sum :', x + y)
+
+
+@dsl.pipeline
+def core() -> int:
+    task = core_comp()
+    task.set_caching_options(False)
+
+    return task.output
+
+
+@dsl.pipeline
+def mantle() -> int:
+    dag_task = core()
+    dag_task.set_caching_options(False)
+
+    return dag_task.output
+
+
+@dsl.pipeline(name=os.path.basename(__file__).removesuffix('.py') + '-pipeline')
+def crust():
+    dag_task = mantle()
+    dag_task.set_caching_options(False)
+
+    task = crust_comp(x=2, y=dag_task.output)
+    task.set_caching_options(False)
+
+
+if __name__ == '__main__':
+    Compiler().compile(
+        pipeline_func=crust,
+        package_path=f"{__file__.removesuffix('.py')}.yaml")
+    client = Client()
```

Comment:
nit: This is a bit unconventional to include a submission code in the execution body, not sure users would intend to run it remotely.

Reply:
The rationale behind this was that it improves the developer experience by making it really easy to kick off a **single** test run when troubleshooting without having to manipulate the contents of `sample_test.py`. I don't think there's any other context in which these modules are run as main. Let me us know if that makes sense / sounds reasonable. If not, we can remove it.